src/dynamo0p3_builtins_mod.f90

```diff
@@ -1,20 +1,34 @@
 !-------------------------------------------------------------------------------
-! (c) The copyright relating to this work is owned jointly by the Crown, 
-! Met Office and NERC 2015. 
-! However, it has been created with the help of the GungHo Consortium, 
+! (c) The copyright relating to this work is owned jointly by the Crown,
+! Met Office and NERC 2015.
+! However, it has been created with the help of the GungHo Consortium,
```

Comment:
Please add "Modified I. Kavcic Met Office " at the appropriate place as you have done for other files.